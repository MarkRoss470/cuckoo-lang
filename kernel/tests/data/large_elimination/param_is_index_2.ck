data Nat : Type where
  | zero : Nat
  | succ : Nat -> Nat

data SomeProp : Nat -> Nat -> Prop where
  | c : (m : Nat) -> (n : Nat) -> SomeProp n m

def some_prop_rec : Prop := SomeProp.rec