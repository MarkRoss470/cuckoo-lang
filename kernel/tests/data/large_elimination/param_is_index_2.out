ERROR at tests/data/large_elimination/param_is_index_2.ck 8:29-8:41: SomeProp.rec takes 1 level argument(s), but 0 were provided.